test/e2e/apiextensions_compositions_legacy_test.go

```diff
@@ -0,0 +1,221 @@
+/*
```

Comment:
None of the tests in this file are new - they just moved into this file and were renamed.